tests/unit/test_kwargs/test_multipart_data.py

```diff
@@ -101,19 +103,17 @@ def test_method(data: t_type = body) -> None:  # type: ignore
 def test_request_body_multi_part_mixed_field_content_types() -> None:
     person = PersonFactory.build()
 
-    class MultiPartFormWithMixedFields(BaseModel):
-        class Config(BaseConfig):
-            arbitrary_types_allowed = True
-
+    @dataclass
+    class MultiPartFormWithMixedFields:
         image: UploadFile
-        tags: List[str]
+        tags: List[int]
```

Comment:
Why the change? Is this not backwards compatible for some reason?

Reply:
it is not. 

see: https://github.com/jcrist/msgspec/issues/477